Doc/library/sqlite3.rst

```diff
@@ -1617,8 +1652,8 @@ The object passed to *protocol* will be of type :class:`PrepareProtocol`.
 .. literalinclude:: ../includes/sqlite3/adapter_point_1.py
 
 
-Registering an adapter callable
-"""""""""""""""""""""""""""""""
+Register an adapter callable
+""""""""""""""""""""""""""""
```

Comment:
As mentioned above, without prefix and continuous form, it sounds like a step that must be executed.
```suggestion
How to register adapter callables
"""""""""""""""""""""""""""""""""
```

Reply:
I agree with this as well, for what its' worth

Reply:
Ok, I'm fine with it. Apply at will :)